### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jeremyr8r**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# github-slack-testing
\ No newline at end of file
+# github-slack-testing
+
+less than an hour to closing time
```
Comment (jeremyrader) [inline README.md]: you must really want to go home

Comment (jeremyr8r) [inline README.md]: yup

**Comment by jeremyr8r** [comment]
this is a regular comment

**Comment by jeremyrader** [comment]
and another one

**Pull request closed without merging**